generated/tests/server/models/user-test.js

```diff
@@ -2,10 +2,10 @@ var sinon = require('sinon');
 var expect = require('chai').expect;
 
 var Sequelize = require('sequelize');
-var dbURI = 'postgres://localhost:5432/testing-fsg';
-var db = new Sequelize(dbURI);
 
-require('../../../server/db/models/user')(db);
+//testing environment variable 
+process.env.NODE_ENV = 'testing';
```

Comment:
We could do this part in the `"testServerJS"` gulp task, just so that it doesn't need to be included in every single test file.